brainzutils/metrics.py

```diff
@@ -29,172 +29,51 @@ def decorated(*args, **kwargs):
 
 @cache.init_required
 @metrics_init_required
-def increment(metric_name, amount=1):
-    """Increment the counter for a metric by a set amount. A metric is a counter that can increment over time
-    and can be used for monitoring any statistic.
-
-    If incrementing the counter causes it to go over redis' internal counter limit of 2**63-1, the counter
-    is reset to 0. The metric name ``tag`` is reserved and cannot be used.
-
-    Arguments:
-        metric_name: the name of a metric
-        amount: the amount to increase the counter by, must be 1 or greater (default: 1)
-
-    Raises:
-        ValueError if amount is less than 1 or greater than 2**63-1
-        ValueError if the reserved metric name ``tag`` is used
+def set(metric_name: str, tags: Dict[str,str]={}, timestamp: int=None, **fields):
+    """
+        Submit a metric to the MetaBrainz influx datastore for graphing/monitoring
```

Comment:
We should add a comment mentioning that the data is tored in InfluxDB line protocol format.